modules/ROOT/pages/database-administration/standard-databases/seed-from-uri.adoc

```diff
@@ -200,7 +215,7 @@ Where `accessKey` and `secretKey` are provided by AWS.
 
 | `file:`
 | `FileSeedProvider`
-| `\file://tmp/backup1.backup`
+| `\file:/tmp/backup1.backup`
```

Comment:
Perhaps, you mean:
```suggestion
| `file:///tmp/backup1.backup`
```

Reply:
No, what I wrote is correct. What you wrote is also correct. Both work. I wrote that one because triple slashes are more cumbersome that single slashes.
The only thing that doesn't work is the one that's currently documented :sweat_smile:

Reply:
I went for a double slash just for consistency with the other URIs.